<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>971</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>House Price Prediction</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Select data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Select">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>751</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PickData">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pick Dataset</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>871</width>
      <height>321</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Avg. Area Income</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Avg. Area House Age</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Avg. Area Number or Rooms</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Avg. Area Number of Bedrooms</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Area Population</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Original Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Predicted Price</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>110</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Coefficient</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Coeff">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>431</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Coefficient</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>370</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Mean Absolute Error (MAE)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>410</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Mean Square Error (MSE)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>450</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Root Mean Square Error (RMSE)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MAE">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>372</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MSE">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RMSE">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>450</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveModel">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>510</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoadModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Avg. Area Income</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Avg. Area House Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Avg. Area Number of Rooms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Avg. Area Number of Bedrooms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>650</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Area Population</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>630</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Predicted Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Income">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_HouseAge">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Rooms">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>570</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Bedrooms">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>610</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Population">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>650</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Pred">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>630</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Predict">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>560</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Training Rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TrainingRate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TrainModel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Train Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EvaluateModel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluate Model</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
